app/views/record/_extended_info.html.erb

```diff
@@ -0,0 +1,145 @@
+    <li>
+      Source: <%= @record.eds_source_title %>
+    </li>
+  <% end %>
+
+  <%# TODO this can apparently include HTML and <br />. Nope. %>
+  <% if @record.eds_authors.present? %>
+    <li>
+      <ul>
+        <% @record.eds_authors.each do |author| %>
+          <li>
+            <%= author %>
+          </li>
+        <% end %>
+      </ul>
+    </li>
+  <% end %>
+
+  <% if clean_affiliations.present? %>
+    <li>
+      <ul>
+        <% clean_affiliations.each do |aff| %>
+          <li>
+            <%= aff %>
+          </li>
+        <% end %>
+      </ul>
+    </li>
+  <% end %>
+
+  <% if @record.eds_publication_info.present? %>
+    <li>
+      Publication info: <%= @record.eds_publication_info %>
+    </li>
+  <% end %>
+
+  <%# TODO how do we decide which ISBN to use %>
+  <% if @record.eds_isbn_print.present? %>
+    <li>
+      ISBN: <%= @record.eds_isbn_print %>
+    </li>
+  <% end %>
+
+  <% if @record.eds_isbn_electronic.present? %>
+    <li>
+      ISBN: <%= @record.eds_isbn_electronic %>
+    </li>
+  <% end %>
+
+  <%# TODO what happens with electronic journals %>
+  <% if @record.eds_issn_print.present? %>
+    <li>
+      ISSN: <%= @record.eds_issn_print %>
+    </li>
+  <% end %>
+
+  <% if @record.eds_document_doi.present? %>
+    <li>
+      DOI: <%= @record.eds_document_doi %>
+    </li>
+  <% end %>
+
+  <% if clean_language.present? %>
+    <li>
+      <%= "Language".pluralize(clean_language.count) %>:
+      <%= clean_language.join(',') %>
+    </li>
+  <% end %>
+
+  <% if @record.eds_physical_description.present? %>
+    <li>
+      Physical description: <%= @record.eds_physical_description %>
+    </li>
+  <% end %>
+
+  <% if @record.eds_database_name.present? %>
+    <li>
+      Database: <%= @record.eds_database_name %>
+    </li>
+  <% end %>
+
+  <%# The wireframes also included edition (for books) and serial publication
+      dates and frequency (for serials), but this info isn't exposed by the
+      EBSCO gem, so we're leaving it out. If this makes it hard for users to
+      evaluate documents, we may have to look harder for that information. %>
+</ul>
+
+<h3 class="subtitle3">Subjects</h3>
+<ul>
+  <% @record.eds_subjects.each do |subject| %>
+    <li>
+      <%= link_to(subject, search_prefix + subject, data: {type: "Subject"} ) %>
+    </li>
+  <% end %>
+</ul>
+
+<% if clean_keywords.present? %>
+<h4>Author supplied Keywords:</h4>
+<ul>
+  <% clean_keywords.each do |keyword| %>
+    <li>
+      <%= link_to(keyword, search_prefix + keyword, data: {type: "Subject"} ) %>
+    </li>
+  <% end %>
+</ul>
+<% end %>
+
+<% if @record.eds_abstract.present? %>
+  <h4>Abstract/summary</h4>
+  <%= sanitize raw @record.eds_abstract %>
+<% end %>
+
+<% if @record.eds_notes.present? %>
+  <h4>Notes</h4>
+  <%# Turns out notes in the wild can include HTML markup, which is encoded and
+      rendered as strings. Let's not do that. %>
+  <%= sanitize raw @record.eds_notes %>
```

Comment:
Do you have an example of this? I want to see how `sanitize raw` is working together here. Thx.

Reply:
Sure. `/record/cat00916a/mit.002201820`, `/record/cat00916a/mit.000824532`, `/record/cat00916a/mit.000292841`. These are all examples where the user sees `<br />` as rendered text. Adding `sanitize raw` means they instead see linebreaks.

Reply:
Ah, right. I was thinking "if you sanitize why are we raw-ing" but now I get it. Thanks for helping my brain :)